nomad/structs/structs.go

```diff
@@ -895,6 +906,61 @@ func (u *UpdateStrategy) Rolling() bool {
 	return u.Stagger > 0 && u.MaxParallel > 0
 }
 
+const (
+	// PeriodicSpecCron is used for a cron spec.
+	PeriodicSpecCron = "cron"
+)
+
+// Periodic defines the interval a job should be run at.
+type PeriodicConfig struct {
+	// Enabled determines if the job should be run periodically.
+	Enabled bool
```

Comment:
Do we really need this? If a Job is not destroyed and it has a periodic block in it, can't we just run it periodically?